content/en/docs/_index.md

```diff
@@ -1,3 +1,3 @@
 ---
-title: Documentation
+title: Kubernetes Documentation
```

Comment:
Good change. 👍

Reply:
This change also percolates up to the top navigation menu, changing "Documentation" to "Kubernetes Documentation" up top as well.

On the whole I like this for SEO purposes but, personally I'd like to keep it "Documentation" in the top nav. 

Looking at `header.html`'s implementation, all you'd have to do is add the `linktitle:` frontmatter property here, as such: 

```suggestion
title: Kubernetes Documentation
linktitle: Documentation
```

Reply:
The way this works, the `linktitle` seems to be what's used in the breadcrumb menu too.